app/config/collections.php

```diff
@@ -2212,6 +2452,99 @@
                 'required' => true,
                 'array' => false,
             ],
+            [
+                '$id' => ID::custom('vcsInstallationId'),
+                'type' => Database::VAR_STRING,
+                'signed' => true,
+                'size' => Database::LENGTH_KEY,
+                'format' => '',
+                'filters' => [],
+                'required' => false,
+                'array' => false,
+            ],
+            [
+                '$id' => ID::custom('vcsInstallationInternalId'),
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => Database::LENGTH_KEY,
+                'signed' => true,
+                'required' => false,
+                'default' => null,
+                'array' => false,
+                'filters' => [],
+            ],
+            [
+                '$id' => ID::custom('vcsRepositoryId'),
+                'type' => Database::VAR_STRING,
+                'signed' => true,
+                'size' => Database::LENGTH_KEY,
+                'format' => '',
+                'filters' => [],
+                'required' => false,
+                'array' => false,
+            ],
+            [
+                '$id' => ID::custom('vcsRepositoryDocId'),
+                'type' => Database::VAR_STRING,
+                'signed' => true,
+                'size' => Database::LENGTH_KEY,
+                'format' => '',
+                'filters' => [],
+                'required' => false,
+                'array' => false,
+            ],
+            [
+                '$id' => ID::custom('vcsRepositoryDocInternalId'),
```

Comment:
```suggestion
                '$id' => ID::custom('vcsRepositoryInternalId'),
```

Why do we use `Doc` here?

Reply:
repository = proxied response from git provider
repository document = our internal document connecting function to specific repository ID

Reply:
Renamed to `repositoryId, repositoryInternalId, providerRepositoryId`